data/changelog/ocaml/2023-09-06-ocaml-5.1.0.rc3.md

```diff
@@ -0,0 +1,55 @@
+---
+title: OCaml 5.1.0 - Third Release Candidate
+description: Third Release Candidate of OCaml 5.1.0
+date: "2023-09-06"
+tags: [ocaml, release]
+changelog: |
+  ## Last Second Bug Fixes
+  - [#11284](https://github.com/ocaml/ocaml/issues/11284), +[#12525](https://github.com/ocaml/ocaml/issues/12525): Use compression of entries scheme when pruning mark stack.
+   Can decrease memory usage for some workloads, otherwise should be
+   unobservable.
+   (Tom Kelly, review by Sabine Schmaltz, Sadiq Jaffer and Damien Doligez)
+
+  - [#12486](https://github.com/ocaml/ocaml/issues/12486): Fix delivery of unhandled effect exceptions on s390x
+  (Miod Vallat, report by Jan Midtgaard, review by Vincent Laviron and Xavier
+  Leroy)
+---
+
+Since last week, there were two significant bugs fixed in the OCaml 5.1.0 runtime (one overflow bug, and a stack corruption bug in the s390x port). Since those bug fixes are as small as they are subtle, they were deemed worthy of a release of a third release candidate for OCaml 5.1.0.
```

Comment:
```suggestion
Since last week, there were two significant bugs fixed in the OCaml 5.1.0 runtime (one overflow bug, and a stack corruption bug in the s390x port). Since those bug fixes are as small as they are subtle, they were deemed worthy of a third release candidate for OCaml 5.1.0.
```
Curious if this repetition of "of a release" is necessary. Is "a release" and "a third release candidate" two different things, which is how it reads now, or is "a third release candidate" a more detailed description of the release, which is how my suggestion reads. Choose whichever is accurate! :)

Reply:
This is an interesting point. On the one hand, `release candidate` is a technical term for "the version of the compiler and libraries that we intend to release" and I tend to read `third release candidate for OCaml 5.1.0` as a expanded version number for `OCaml 5.1.0~rc3`, so there is a difference in my mind between a release and a release candidate. On the other hand, a release candidate ought to be released, so the wording "release of a release candidate" is redundant and your suggestion reads better to me. Thanks!

Reply:
My pleasure!